### PR TITLE
Introduction to html

### PR DESCRIPTION
**Pull request by Jonak-Adipta-Kalita**
# We can't let HTML behind!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,6 +314,15 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## HTML
+
+- [HTML Cheat sheet by CodeWithHarry](https://www.codewithharry.com/blogpost/html-cheatsheet)
+- [How to make your HTML responsive by adding a single line of CSS](https://medium.com/free-code-camp/how-to-make-your-html-responsive-by-adding-a-single-line-of-css-2a62de81e431)
```
Comment (shahednasser) [inline README.md:320]: Single blog posts are not allowed so please remove this resource

File: README.md [hunk 2]
```diff
@@ -313,6 +314,15 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [15 HTML element methods you’ve potentially never heard of](https://medium.com/hackernoon/15-html-element-methods-youve-potentially-never-heard-of-fc6863e41b2a)
```
Comment (shahednasser) [inline README.md:321]: This is more of a JavaScript resource, but it's also a single blog post so please remove it.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @Jonak-Adipta-Kalita thank you for your contribution! I have added some comments below. Please note that single blog posts are not allowed (you can check the allowed resources in the [contributing guidelines](https://github.com/shahednasser/awesome-resources/blob/master/CONTRIBUTING.md)), however, the 5 HTML tricks I've allowed as I think a lot of people miss this when it comes to HTML so it can be quite resourceful.

File: README.md (hunk 2 above)
Comment (Jonak-Adipta-Kalita) [reply README.md:321]: Ok

Comment (Jonak-Adipta-Kalita) [reply README.md:320]: Please check if its okay now

**Comment by Jonak-Adipta-Kalita** [comment]
Please check if its good now

**Comment by shahednasser** [comment]
Thank you!

**Comment by Jonak-Adipta-Kalita** [comment]
My pleasure

**Pull request merged**